.vscode/launch.json

```diff
@@ -4,142 +4,162 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "iOS - Preview",
```

Comment:
Not much changes to this file. I just changed the name of the entries to start with the command name first, and also reordered the entries to group the commands together.